src/App.tsx

```diff
@@ -1,25 +1,51 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+
+  const [perPage, setPerPage] = useState(5)
```

Comment:
It's a good practice to specify the type of state when initializing it, even if it can be inferred. This makes the code more readable and maintainable. Consider specifying the type for 'perPage' state.